src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const BASE_PRICE = days * PRICE_PER_DAY;
```

Comment:
Static constants, such as strings or numbers, are called uppercase. 
If the constant is calculated in some way, then it is better to call it camelCase